src/sheet-types.tsx

```diff
@@ -71,7 +74,7 @@ export type SheetType = keyof SheetPropsByType;
 export type SheetResultByType = {
     RADIO_LIST: {action: 'SUBMIT'; selectedId: string} | {action: 'DISMISS'};
     ACTIONS_LIST: {action: 'SUBMIT'; selectedId: string} | {action: 'DISMISS'};
-    INFO: void;
+    INFO: {action: 'DISMISS'};
```

Comment:
I think it's reasonable to always return 'DISMISS' instead of nothing. If the users doesn't want to do anything with it, they can just ignore the result